frontend/packages/noobaa-storage-plugin/src/components/object-data-reduction-card/object-data-reduction-card.tsx

```diff
@@ -14,7 +14,7 @@ import {
 
 import { EfficiencyItem, SavingsItem } from './object-data-reduction-card-item';
 import { getPropsData } from '../../utils';
-import { ObjectDataReductionQueries } from '../../queries';
+import { ObjectDataReductionQueries } from '../../constants/queries';
```

Comment:
need to fix it here since the page is breaking on load!

Reply:
so to make this in parallel with other plugins I have to shift the `queries.ts` file